matRad_rayTracing.m

```diff
@@ -139,3 +141,6 @@
     radDepthV{i} = radDepthCube{i}(V);
 end
 
+radDepthsMat = radDepthCube{1};
```

Comment:
If we want this to generalize to multiple dimensions (4D ct data) do we need to pass on all cubes from the cell array?`

radDepthsMat = radDepthCube;